### ISSUE TITLE
Detects merge conflict prompt strings

### ISSUE THREAD

**Issue by zzzzzzboy** [eslint/eslint#14845]
**Please describe what the rule should do:**
When resolving merge conflicts, You might forget to delete the merge conflict prompt code string, this rule will help you detect these strings.

**What new ECMAScript feature does this rule relate to?**
none

**What category of rule is this? (place an "X" next to just one item)**

[X] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

![image](https://user-images.githubusercontent.com/42526781/127255426-2cc58e42-b1c5-406c-89a0-c7a21602950a.png)
```js
>>>>>>> 2sdwe5sad1551we451sdwe5412ase54asd21we5a
```

**Why should this rule be included in ESLint (instead of a plugin)?**
Not sure

**Are you willing to submit a pull request to implement this rule?**
Not sure

**Comment by aladdin-add** [comment]
it should have been reported by the parser (something like `unexpected char >>>>`).

**Comment by nzakas** [comment]
Yes, this is a parsing error so will already show up.

**Comment by zzzzzzboy** [comment]
> Yes, this is a parsing error so will already show up.

@nzakas @aladdin-add 
In fact, from the time I submitted the code to the time I posted it, no errors were thrown.
As shown below, it can only detect strings like "<<<" and throw an error, but it can't detect" >>>".
![image](https://user-images.githubusercontent.com/42526781/127420984-ea05bc66-e4df-435c-ba14-aa298b976a91.png)

Or what else do I need to configure to detect this error？ Thanks

**Comment by aladdin-add** [comment]
it seems an issue in vue-eslint-parser (as seen in the picture, you are using the vue-eslint-parser, not the default parser).
you can file the issue in vue-eslint-parser repo. :)
